tests/v2/validation/Dockerfile.validation

```diff
@@ -15,7 +15,7 @@ WORKDIR $WORKSPACE/tests/v2/validation
 COPY [".", "$WORKSPACE"]
 
 RUN go mod download && \
-    go get gotest.tools/gotestsum
+    go install gotest.tools/gotestsum@latest
```

Comment:
Is it ok to always grab the latest or do we want to use an specific version for more control?

Reply:
Not in this case, we always just want the latest version of gotestsum, and this package isn't updated regularly. But it's one the most used tools in go testing